### DISCUSSION TITLE
delete bucket too slow

### DISCUSSION THREAD

**Discussion by RichardQin1** [minio/minio#13303] [Q&A]
700W pictures are stored in one bucket
No folders were split.

but now .  
mc rb cannot delete the bucket.  run time >24h.
Who can tell me why ? thank

**Comment by harshavardhana** [comment]
> 700W pictures are stored in one bucket
> No folders were split.
> 
> but now .  
> mc rb cannot delete the bucket.  run time >24h.
> Who can tell me why ? thank

You are not explaining your setup details. Without that how can we help?
